webpack.config.dev.js

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Det blir ikke not annerledes en tidligere. Hovedgrunnen er for å slippe en stor fil med masse conditonals på DEBUG. Se [her].(https://github.com/NDLANO/learningpath-frontend/blob/a22144dd5879fb442b5c02f7e55ec82dd0afab82/webpack.config.js)

Det er script'ene i [pacakge.json](https://github.com/NDLANO/learningpath-frontend/blob/5fbced16161ecbfb340f4bd7574139847135c687/package.json) som bestemmer hva som skal bygges. For prod kjøres `npm run build `for å bygge en optimalisert versjon av frontend ressurser, mens `npm run start-prod` kjører opp en node server som bruker de optimaliserte ressursene (les CMD i Dockerfile).

`npm start` starter en local server med dev configen som inkluderer ting som hot reloading, sourcemaps, etc.